src/main/deploy/choreo/ThreeL4ProcessorLolipop/Fifth Path.traj

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
why are these trajs named this. use the same level of descriptiveness that the other autos use